molo/yourwords/templatetags/competition_tag.py

```diff
@@ -0,0 +1,17 @@
+from django import template
+from copy import copy
+from molo.yourwords.models import YourWordsCompetition
+
+register = template.Library()
+
+
+@register.inclusion_tag(
+    'yourwords/your_words_competition_tag.html',
+    takes_context=True
+)
+def your_words_competition(context, page=None):
+    context = copy(context)
+    context.update({
+        'competitions': YourWordsCompetition.objects.live()
```

Comment:
@miltontony when I add ` .child_of(page) ` I get ` 'NoneType' object has no attribute 'path' ` can you point out where I'm going wrong there?